cosmwasm/contracts/crosschain-registry/src/state.rs

```diff
@@ -0,0 +1,21 @@
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::Addr;
+use cw_storage_plus::{Item, Map};
+
+// CONTRACT_ALIAS_MAP is a map from a contract alias to a contract address
+pub const CONTRACT_ALIAS_MAP: Map<&str, String> = Map::new("contract_alias_map");
+
+// CHAIN_TO_CHAIN_CHANNEL_MAP is a map from a source<>destination chain pair to its respective channel id
+pub const CHAIN_TO_CHAIN_CHANNEL_MAP: Map<(&str, &str), String> =
+    Map::new("chain_to_chain_channel_map");
+
+// CHANNEL_TO_CHAIN_CHAIN_MAP is a map from a channel id on a source chain to its respective destination chain
+pub const CHANNEL_ON_CHAIN_CHAIN_MAP: Map<(&str, &str), String> =
+    Map::new("channel_to_chain_chain_map");
```

Comment:
These strings are the storage prefix. We're not storing a lot of data here so it may not matter that they're long, but something to keep in mind for contracts with a lot of data.

An alternaative is to create an enum or a const that maps to short strings.

Reply:
I think this is what you were describing!

https://github.com/osmosis-labs/osmosis/pull/4320/commits/d109bd04ceff1de8c67b4a5048ab3793ee3cad5e